### DISCUSSION TITLE
Compatibility matrix

### DISCUSSION THREAD

**Discussion by ddehen** [redisson/redisson#4894] [Q&A] [answered]
Is there a compatibility matrix available? How to know which Redisson client version is compatible with a specific Redis engine version?

**Comment by Milk-Bottle** [comment]
Yes, Redisson provides an official compatibility matrix between Redisson client versions and Redis engine versions. You can find this information in the Redisson documentation, which includes the Redis engine versions supported by each Redisson version, as well as the minimum Redisson version required for each Redis engine version.

You can find this information on the "Version History" page in the Redisson documentation. On that page, you can select the desired Redisson version and view the Redis engine versions supported by that version.

**Reply by mrniko** [reply]
Is it a ChatGPT answer too? :D (joke)

**Reply by Milk-Bottle** [reply]
Sorry, I didn't mark the source of the answer. If the answer is wrong, I can delete the comment

**Comment by mrniko** [comment, marked as answer]
You can use current version with any version of Redis starting from 3.0. Any Redisson version can be run with Redis 3.0 and higher. The only limitation is ability to use some features represented by Redisson API which might be compatible with Redis 4, 5, 6.2 or 7

**Reply by ddehen** [reply]
Thanks!
